lib/src/localizations/messages_all.dart

```diff
@@ -8,7 +8,7 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_en.dart' deferred as messages_en;
```

Comment:
When I try to enable hot reloading without this change I get the following message:

> Reloading support for deferred loading has not yet been implemented: library 'package:flutter/src/widgets/placeholder.dart' has deferred import 'messages_en'

Reply:
Thanks for the heads up! Hadn't seen that, but I'll take a deeper look since this file is generated by the the i18n tools.